tokens/properties/components/Dropdown.json

```diff
@@ -0,0 +1,221 @@
+{
+    "wikit-Dropdown": {
+        "default-width": {
+            "value": "{dimension.width.full.value}"
+        },
+        "border-color": {
+            "value": "{border.color.base.default.value}"
+        },
+        "border-style": {
+            "value": "{border.style.base.value}"
+        },
+        "border-width": {
+            "value": "{border.width.thin.value}"
+        },
+        "border-radius": {
+            "value": "{border.radius.base.value}"
+        },
+        "background-color": {
+            "value": "{background.color.neutral.default.value}"
+        },
+        "font-family": {
```

Comment:
While implementing, I noticed that it makes probably more sense to group these font-familiy and line-height styles into a "selectedOption" subsection to better contrast them with the placeholder styles.

Reply:
Makes sense! For a moment, I wanted to suggest grouping both (placeholder and selected option) under the same object and call it "dropdown text" or similar, since they share exactly the same styles. What stopped me was 1. the will to keep a certain parallelism between field elements, 2. the thought that maybe their styles could be updated and differ in the future (not so sure about this, though).